server/lib/src/be/idl_arc_sqlite.rs

```diff
@@ -58,6 +59,7 @@ pub struct IdlArcSqlite {
     op_ts_max: CowCell<Option<Duration>>,
     allids: CowCell<IDLBitRange>,
     maxid: CowCell<u64>,
+    write_ops_since_last_repl: CowCell<u64>,
```

Comment:
This shouldn,t be here, it should be in the QueryServer instead.

Reply:
sure!